### PR TITLE
Gp 2022.10.22

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 7/07_01_CIEL07_1_3.txt [hunk 1]
```diff
@@ -487,7 +487,7 @@ In one smooth motion, I uncross my legs and sidle over to Senpai.
 [sha:d4c269e4649982014ce2c38666eb68ff296c2dd1]{
 -- Page 128, Offset 15112.
 -- 「なぜ!?　爽やかに気が狂ってます!?」
-// Debonair delirium?
+// Serene insanity?
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_1_3.txt:490]: lmao i prefer this

**Review by Avuxo — APPROVED** [review]
nice work today fellas

File: script/Ciel/Day 7/07_01_CIEL07_7.txt [hunk 2]
```diff
@@ -471,7 +471,7 @@ Mere seconds ago, a web of over seven hundred spirit strings had surrounded the
 "I didn't hear any Magecraft Incantation...
  You fuckin' burn 'em all up just by pouring magical energy through your fingertips?
  You're a real goddamn dynamo.
- Just how many hundreds of mages worth of energy can you generate?
+ Just how many hundreds a mages worth of energy can you generate?
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_7.txt:474]: This one's on the border of typo imo, I'd probably keep original.

File: script/Ciel/Day 7/07_01_CIEL07_7.txt [hunk 3]
```diff
@@ -488,7 +488,7 @@ Well, guess there's no way you'd be able to hold onto two or three Idea Bloods i
 -- Page 615, Offset 16286.
 -- 「待てやめろ、転落死なんてマヌケな死に方は冗談でも勘弁だ、ってもう燃えてる燃えてる、相変わらず容赦ねえなテメェ！
 "Whoa, stop, falling to my death wouldn't even be a funny way to go――ah shit, it's already on fire!
- As fuckin' merciless as ever, aintcha?
+ As fuckin' merciless as ever, aintcha?!
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_7.txt:491]: ain'tcha?

File: script/Ciel/Day 7/07_01_CIEL07_3.txt [hunk 4]
```diff
@@ -472,7 +472,7 @@ If Arcueid is still here, then that must mean there are still vampires left to d
 // P: allowed to go any further by the Ancestors. (Ancestor permission is required to ascend beyond Rank VI).
 // P: That said, I'm fairly certain this isn't referring to Roa but is rather actually a generic statement
 "Yep, I'm here to deal with the petty Dead Apostles that remain.
- This doesn't have anything to do with Vlov, though."
+ Not that it has a lot to do with Vlov, though."
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_3.txt:475]: Not that it has much to do with Vlov, though.

File: script/Ciel/Day 7/07_01_CIEL07_1_3.txt [hunk 5]
```diff
@@ -337,7 +337,7 @@ Senpai is looking away awkwardly. It seems she's realized what I'm hinting at.
  It's just that I need a little more time to make it.
  Yesterday I felt like I'd lost my touch,
  and I know you said that taste isn't the most important factor,
- but I really do think that it's important for food to be delicious!"
+ but I really do think that it's important that food be delicious!"
```
Comment (yk1512) [inline script/Ciel/Day 7/07_01_CIEL07_1_3.txt:340]: prefer original phrasing here

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 7/07_01_CIEL07_1B.txt [hunk 6]
```diff
@@ -58,22 +57,23 @@ There are no more vampires. Arcueid and Senpai are both gone.
 [sha:fb80bdba79ccee84e99ed8325d669beda533ff24]{
 -- Page 35, Offset 14893.
 -- 残ったものは何の変哲もない今まで通りの、
-All that remains is the routine, everyday―――
+All that remains is the routine, every―――
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_1B.txt:60]: The interruption should be more obvious.
everyda――― or something

File: script/Ciel/Day 7/07_01_CIEL07_1B.txt [hunk 7]
```diff
@@ -28,15 +28,15 @@ I can feel myself crack a smile.
...
+I can't help but feel this way whenever I think about her.
  I guess it's because she was such an incredible character.
 }
 [sha:5a010c24a17359803d800f2bcd3a05e8e2c2126a]{
 -- Page 34, Offset 14889.
 -- 無邪気に笑ったり怒ったり、たまにとんでもなく怖かったりもしたけど、基本的にはいいヤツだった。
 There was an innocence in the way that she got happy or angry,
  and there were times when she was truly terrifying,
- but she was fundamentally a good person.
+ but she was a fundamentally good person.
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_1B.txt:39]: Looking over this again, it actually feels worse.
Rather use the version with commas
>but she was, fundamentally, a good person.

**Pull request merged**